tests/system/HTTP/Files/FileCollectionTest.php

```diff
@@ -55,6 +56,12 @@ public function testAllReturnsValidSingleFile()
 
         $this->assertSame('someFile.txt', $file->getName());
         $this->assertSame(124, $file->getSize());
+
+        if (version_compare(PHP_VERSION, '8.1', '>=')) {
```

Comment:
Because we cannot upload in testing and `$_FILES` is given, we cannot write test like this.
Just add another test with `$_FILES` without `full_path` is enough.

Reply:
Yes. Just updated that. I realized how stupid that was.